plugins/json/src/main/java/org/apache/struts2/json/JSONUtil.java

```diff
@@ -410,7 +411,9 @@ else if (serializationParams.getErrorCode() > 0)
     public static Method[] listSMDMethods(Class clazz, boolean ignoreInterfaces) {
         final List<Method> methods = new LinkedList<>();
         if (ignoreInterfaces) {
-            for (Method method : clazz.getMethods()) {
+            Method[] SMDMethods = clazz.getMethods();
```

Comment:
Could you use `methods` instead of `SMDMethods`?

Reply:
I think methods is used at 412 `final List<Method> methods = new LinkedList<>();`